src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -136,14 +139,41 @@ public String getProjectName() {
         return projectName;
     }
 
+    /**
+     * Sets directory suffix.
+     *
+     * @param directorySuffix directory suffix
+     */
+    public void setDirectorySuffix(String directorySuffix) {
+        this.directorySuffix = directorySuffix;
+    }
+
+    /**
+     * Returns directory suffix
+     *
+     * @return directory suffix
+     */
+    public String getDirectorySuffix() {
+        return StringUtils.defaultString(directorySuffix);
+    }
+
+    /**
+     * Returns directory suffix with separator prepended if necessary
+     *
+     * @return directory suffix with prepended separator
+     */
+    public String getDirectorySuffixWithSeparator() {
+        return StringUtils.isEmpty(directorySuffix) ? "" : SUFFIX_SEPARATOR + directorySuffix;
```

Comment:
can't find unit test for new methods

Reply:
Ah! good point - missed that. I added the unit tests now.